src/components/deliver-address-card/index.jsx

```diff
@@ -1,35 +1,128 @@
-export function renderDeliveryAddressCard({name, address, pincode}) {
+import { useState } from "react";
+
+export function renderDeliveryAddressCard() {
     return function DeliveryAddressCard() {
+        const [headBackground, setHeadBackground] = useState("bg-blue-500");
```

Comment:
cwe can change this to instead have a single address object. and give each of the form fields to trigger updating the state for the specific property. that way we dont need to even perform data accumulation aint the handleSave function